python/kserve/kserve/model_server.py

```diff
@@ -360,12 +341,17 @@ def default_exception_handler(
         """Default exception handler for event loop.
 
         This is called when an exception occurs and no exception handler is set.
-        By default, this will shut down the server gracefully.
-
         This can be called by a custom exception handler that wants to defer to the default handler behavior.
         """
-        # gracefully shutdown the server
-        loop.run_until_complete(self.stop())
+        if "exception" in context:
+            logger.error(f"Caught exception: {context.get('exception')}")
+        logger.error(f"message: { context.get('message')}")
+        if "future" in context:
+            future = context["future"]
+            if future.done():
+                future_exception = future.exception()
+                if future_exception:
+                    logger.error(f"Future exception: {future_exception}")
```

Comment:
what's the difference between exception and future_exception?

Reply:
It turns out both the exceptions are the same. Removed